gc/base/standard/ConcurrentSweepScheme.cpp

```diff
@@ -1530,6 +1529,117 @@ omrtty_printf("]");
 }
 
 /**
+ * Due to heap resize at the end of global gc(after sweepForMinimumSize),
+ * some concurrent sweepStates (SweepChunk, heapSizeToConnect, heapSizeConnected, ApproximateFree...) need to be updated
+ * without this updating, the free memory, which is shifted from LOA to SOA, would be lost until the next global gc and memory pool reportings
+ * in verbose gc and mxbean are wrong.
+ * TODO:
+ */
+void
+MM_ConcurrentSweepScheme::updateSweepStates(MM_EnvironmentBase* env, uintptr_t resizeType)
```

Comment:
In general how long does this function take to complete? How many sweep chunks are there?